### PR TITLE
update csvtk

### PR DESCRIPTION
**Pull request by shenwei356**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/csvtk/meta.yaml [hunk 1]
```diff
@@ -12,17 +12,17 @@ build:
   skip: False
 
 source:
-  fn: csvtk_v0.4.5_osx.tar.gz # [osx]
-  url: https://github.com/shenwei356/csvtk/releases/download/v0.4.5/csvtk_darwin_amd64.tar.gz # [osx]
-  md5: 9fe6e0203ab9563d2c61985668cb530d # [osx]
+  fn: csvtk_v0.4.6_osx.tar.gz # [osx]
```
Comment (bgruening) [inline recipes/csvtk/meta.yaml]: You could use macros to make such updates easier and less error prone.

Comment (shenwei356) [inline recipes/csvtk/meta.yaml]: Thanks. How it can be achieved?

Comment (bgruening) [inline recipes/csvtk/meta.yaml]: have a look at: https://github.com/bioconda/bioconda-recipes/blob/65d44ad2de2cc662efd8371a712c83083587bb17/recipes/clustalo/meta.yaml#L1

Comment (shenwei356) [inline recipes/csvtk/meta.yaml]: thanks a lot

**Pull request merged**